lacommunaute/forum_moderation/utils.py

```diff
@@ -1,25 +1,29 @@
+import re
+
 from django.conf import settings
 from langdetect import detect
-from machina.models.fields import render_func
-from markdown2 import Markdown
 
 from lacommunaute.forum_moderation.models import BlockedDomainName, BlockedEmail
 
 
+def contains_html(text):
+    html_regex = r"<[^>]+>"
```

Comment:
:wolf: 

```
<img src="http://upload.wikimedia.org/wikipedia/commons/7/75/Cute_grey_kitten.jpg" onclick="alert('XSS');"`
```

Note les warnings dans la console que la CSP a bloqué le chargement et du handler `onclick` et de l’image. Il suffit de bypasser la CSP (ou d’une erreur de config CSP) et c’est XSS.

Reply:
rolled back, en forçant le safe_mode lors du rendering dans `check_post_approbation`